ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/execution/StoreBackedCommandExecutor.scala

```diff
@@ -45,8 +45,11 @@ final class StoreBackedCommandExecutor(
   )(
       implicit ec: ExecutionContext,
       logCtx: LoggingContext,
-  ): Future[Either[ErrorCause, CommandExecutionResult]] =
-    consume(commands.submitter, engine.submit(commands.commands, participant, submissionSeed))
+  ): Future[Either[ErrorCause, CommandExecutionResult]] = {
+    val start = System.nanoTime()
+    val submissionResult = engine.submit(commands.commands, participant, submissionSeed)
+    val interpretationTimeNanos = System.nanoTime() - start
```

Comment:
I don't think you're measuring the interpretation time here. `engine.submit` returns a continuation that the `consume` call below will actually fill. I'd stop the timing after `Blinding.checkAuthorizationAndBlind`.

Reply:
Good point -- changed accordingly.